Training/Easy/defibrillators.rb

```diff
@@ -26,11 +26,11 @@ def parse(var)
     (var.sub(',', '.').to_f * Math::PI) / 180
   end
 
-  def find_closest(defibrilators_hash)
-    min = defibrilators_hash.values.first
-    closest_name = defibrilators_hash.keys.first
+  def find_closest(defibrillators_hash)
```

Comment:
Instead of writing a method yourself, you can use Enumerable#min_by:
```ruby
closest_name, closest_value = defibrillators_hash.min_by { |pair| pair.last }
```
Also, if you're not going to use the second value, you can write it like this:
```ruby
closest_name, _ = defibrillators_hash.min_by { |pair| pair.last }
```